linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/HttpConfigTest.scala

```diff
@@ -113,7 +113,7 @@ class HttpConfigTest extends FunSuite with Awaits {
                   |- port: 5000
       """.stripMargin
     val config = parse(yaml)
-    val identifier = config.routerParams[Http.param.HttpIdentifier]
+    val identifier = config.routerParams(Stack.Params.empty)[Http.param.HttpIdentifier]
       .id(Path.read("/svc"), () => Dtab.empty)
     val req = Request(Method.Get, "/one/two/three")
     assert(
```

Comment:
Should we add a test that validates that we are receiving a `StatsReceiver` in an identifier to show that it propagates through initialization?